recipes/pyyaml/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "3.11" %}
+
+
+package:
+  name: pyyaml
+  version: {{ version }}
+
+source:
+  fn: PyYAML-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/P/PyYAML/PyYAML-{{ version }}.tar.gz
+  md5: f50e08ef0fe55178479d3a618efe21db
+
+build:
+  number: 0
+  script:
+    - python setup.py --with-libyaml build_ext --include-dirs="${PREFIX}/include" --library-dirs="${PREFIX}/lib"  # [unix]
+    - python setup.py --with-libyaml build_ext --include-dirs="%LIBRARY_INC% --library-dirs=%LIBRARY_LIB%         # [win]
```

Comment:
I am mystified.  I think the main reason is that it is indeed a setting of INCLUDE going wrong (setenv.cmd replacing it), but the others set INCLUDE correctly (and the --include-dirs is superfluous there).  I'm guessing it is only strictly necessary for the VS 2010 build?  Wild conjecture here.